CMakeLists.txt

```diff
@@ -10,6 +10,28 @@ if(POLICY CMP0076)
     cmake_policy(SET CMP0076 NEW)
 endif()
 
+add_executable(${APP_TARGET})
+
+# Create Mbed OS library
+add_library(mbed-os OBJECT)
+
+include(${MBED_ROOT}/../.mbedbuild/mbed_config.cmake)
```

Comment:
If tools generate this, should they tell us how to access it ? Rather than having this as it depends on two different things (mbed root and then placement of config coming from tools). If tools generate it, they should let us know where it is and we use the path to access config.

Reply:
Generating `MBED_CONFIG_PATH` from the tools makes sense to me.

Reply:
I agree. Do we need to change something in this PR?